http-api/Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /usr/src/radicle-client-services
 COPY . .
 
 WORKDIR /usr/src/radicle-client-services/http-api
-RUN cargo install --locked --path .
+RUN cargo install --all-features --locked --path .
```

Comment:
Not sure I'd want this as the default, since it won't work in non-gcp contexts, right?

Reply:
It won't cause any failures.  Logging is going to work as if gcp feature didn't exist.

Reply:
Ah sweet.